docs/book/src/topics/custom-vnet.md

```diff
@@ -63,13 +63,13 @@ If no CIDR block is provided, `10.0.0.0/8` will be used by default, with default
 
 Whenever using custom vnet and subnet names and/or a different vnet resource group, please make sure to update the `azure.json` content part of both the nodes and control planes' `kubeadmConfigSpec` accordingly before creating the cluster.
 
-### Custom Ingress Rules
+### Custom Security Rules
```

Comment:
Should we add a note about ingress rules being renamed to security rules somewhere?